src/runtime_src/xdp/profile/database/static_info_database.h

```diff
@@ -110,14 +110,15 @@ namespace xdp {
     std::mutex aieLock ;
     std::mutex appStyleLock;
     std::mutex hwCtxImplUIDMapLock;
+    std::mutex aieProfileConfigLock; 
+    mutable std::mutex aieMetadataReaderLock; 
```

Comment:
If we use "aieMetadataReaderLock" to fetch only in getAIEMetadataReader, do we still need "mutable" ?